src/components/WeeklyCalendar/week/day.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useEventsOfDay } from './hooks';
+import Event from './event';
+import StyledDay from './styled/day';
+
+export default function Day(props) {
+    const { day, events, onEventClick, locale } = props;
+    const eventsOfDay = useEventsOfDay(events, day);
+    const Events = () =>
+        eventsOfDay.map((event, index) => (
+            // eslint-disable-next-line react/no-array-index-key
+            <Event key={index} onEventClick={onEventClick} locale={locale} event={event} />
```

Comment:
generating unique key will help with perf :)

Reply:
if at some point the <Event /> can receive the props of the event not as an object that will help with perf as well, preferred props values are primitives that are reactive by excellence

Reply:
can you do this <Event {...event} /> ??